.cargo/config.toml

```diff
@@ -13,6 +13,10 @@ rustflags = [
     "-Clink-arg=-Wl,--no-rosegment", #4574
 ]
 
+# Remove this if your x86_64 macos does not support AVX2 vector extensions.
+[target.x86_64-apple-darwin]
+rustflags = ["-Ctarget-feature=+avx2"]
+
 [target.aarch64-apple-darwin]
```

Comment:
Does it mean that `aarch64-apple` implies SIMD?

Reply:
> Does it mean that `aarch64-apple` implies SIMD?

I think it is `neon` by default on `aarch64-apple`

Reply:
> Remove this if your x86_64 macos does not support AVX2 vector extensions.

I think these comments are not right, we cannot compile to target without SIMD now, comments here should inform the developers about the requirements for SIMD instead.